certified-connectors/Mobsim Send SMS/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+  {
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Chave de API",
+          "description": "O Chave de API para esta API",
```

Comment:
@jpjsantos please make sure that all summaries, x-ms-summaries, and descriptions are in English and are grammatically correct and free of typographical errors. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).